tests/e2e/specs/devfiles/EmptyWorkspace.spec.ts

```diff
@@ -1,30 +1,33 @@
 /*********************************************************************
- * Copyright (c) 2020 Red Hat, Inc.
+ * Copyright (c) 2020-2023 Red Hat, Inc.
  *
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
-import { e2eContainer } from '../../../inversify.config';
+import { e2eContainer } from '../../configs/inversify.config';
 import { ActivityBar, ViewControl, Workbench } from 'monaco-page-objects';
-import { CLASSES } from '../../../inversify.types';
-import { WorkspaceHandlingTests } from '../../../testsLibrary/WorkspaceHandlingTests';
-import { Logger } from '../../../utils/Logger';
-import CheReporter from '../../../driver/CheReporter';
-import { ProjectAndFileTestsCheCode } from '../../../testsLibrary/che-code/ProjectAndFileTestsCheCode';
+import { CLASSES } from '../../configs/inversify.types';
+import { WorkspaceHandlingTests } from '../../tests-library/WorkspaceHandlingTests';
+import { Logger } from '../../utils/Logger';
+import { ProjectAndFileTests } from '../../tests-library/ProjectAndFileTests';
+import { LoginTests } from '../../tests-library/LoginTests';
+import { registerRunningWorkspace } from '../MochaHooks';
```

Comment:
Are static imports like these optimal?

Reply:
interesting question.. what is your opinion?

Reply:
In my opinion it could cause complications when the class is intended to be imported in a constructor, but overall I'm indifferent. It just that in-code it's not easily visible whether you're importing a class or a method.

Also, potential conflicts if multiple classes have a method with the same name.

Reply:
I don`t think it is a good idea to create instance of MochaHooks in every test suite file. Let`s keep is static?

Reply:
agreed, but we can still import MochaHooks and call it MochaHooks.registerRunningWorkspace or can we not?

Reply:
it is not a Class it is just function

Reply:
oh, I see, thanks for the explanation